gamestonk_terminal/cryptocurrency/crypto_controller.py

```diff
@@ -0,0 +1,107 @@
+__docformat__ = "numpy"
+
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+from prompt_toolkit.completion import NestedCompleter
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.cryptocurrency import coin_api
+
+
+class CryptoController:
+
+    CHOICES = ["help", "q", "quit", "load", "view", "clear"]
+
+    def __init__(self):
+
+        self.crypto_parser = argparse.ArgumentParser(add_help=False, prog="crypto")
+        self.crypto_parser.add_argument("cmd", choices=self.CHOICES)
+        self.current_coin = None
+        self.current_df = pd.DataFrame()
+
+    @staticmethod
+    def print_help(current_coin):
+        """ Print help """
+        print("\nCryptocurrency:")
+        print("   help          show this menu again")
+        print("   q             quit this menu, and shows back to main menu")
+        print("   quit          quit to abandon program")
+        print(f"\nCurrent Coin : {current_coin}")
+        print("")
+        print("   load          load cryptocurrency data")
+        print("   view          load and view cryptocurrency data")
+        print("   clear          load cryptocurrency data")
```

Comment:
I think the descriptions are wrong here! And there's an extra space after  clear

Reply:
yup I am seeing that now